app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  before_save { self.email = email.downcase }
```

Comment:
```suggestion
  before_save { email.downcase! }
```
phân biệt giúp a: `.downcase` và `.downcase!` nhé